sdlib/d/gc/hpd.d

```diff
@@ -111,6 +111,46 @@ public:
 		return bestIndex;
 	}
 
+	bool reserve(uint index, uint pages) {
```

Comment:
It doesn't count as an allocation, so you clearly can't call this reserve. This is the analogous of `clear`, s maybe `set`?